src/mangata/mangata.js

```diff
@@ -110,6 +110,7 @@ async function main() {
         schedule,
         mangataHelper.config.paraId,
         0,
+        { V1: { parents: 1, interior: { X1: { Parachain: mangataHelper.config.paraId } } } },
```

Comment:
In Rococo environment, this `mangataHelper.config.paraId` is the Mangata paraId 2110.